### ISSUE TITLE
SPro subvision 155+ incompatible

### ISSUE THREAD

**Issue by RITCHIEHuang** [ALIZE-Speaker-Recognition/android-alize#16]
Can you provide the subvision of 155+ SPro ?

**Comment by AhmadKhattak** [comment]
Hi, 

First, download the spro-5.0 from here 

[https://gforge.inria.fr/frs/?group_id=532](https://gforge.inria.fr/frs/?group_id=532)

use the following command in terminal to get the spro latest revision from subversion,

`svn checkout svn://scm.gforge.inria.fr/svnroot/spro/
`
Actually, the revision 155 consists of the following

https://gforge.inria.fr/scm/browser.php?group_id=532

and similarly you can check for revision 156 and 157.

So, basically what I did was copy the files of revision 155+ from the spro/trunk/ directory that I got from subversion into the folder of the spro-5.0 and followed the steps to install spro.

**Comment by RITCHIEHuang** [comment]
I will try that, thanks.

**Comment by tevamerlin** [comment]
Hi,

In order to make it easier for everybody to get the right version of SPro, revision 5.0-157 is now downlodable directly from ALIZÉ’s website:
http://alize.univ-avignon.fr/spro-5.0-157.tar.gz
